modules/portworx/examples/simple/main.tf

```diff
@@ -0,0 +1,44 @@
+# TEMPLATE: This file was automatically generated with `generate_addon_structure.sh`
+# TEMPLATE: and should be modified as necessary
+# TEMPLATE:
+# TEMPLATE: All ".tf" files are parsed at once. There is no benefit to numerically prefixed filenames. Keep all resource definitions in "main.tf".
+# TEMPLATE:
+# TEMPLATE: When main.tf becomes unwieldy, consider submodules (https://www.terraform.io/docs/language/modules/develop/structure.html) 
+# TEMPLATE: and dependency inversion (https://www.terraform.io/docs/language/modules/develop/composition.html).
+#
+provider "equinix" {
+  auth_token = var.metal_auth_token
+}
+
+module "equinix_kubernetes_addons" {
+  source = "../../../../"
+
+  equinix_project = var.metal_project_id
+  equinix_metro   = "SV"
+
+  ssh_host        = var.host
+  ssh_private_key = file(var.private_key_path)
```

Comment:
You can safely remove from the example all those attributes that do not apply to this addon

```suggestion
```